src/hooks/data/useRunnerData.ts

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from "react";
+import { SignedRunnersResponse } from "../../types";
+import useSwr from "../useSwr";
+
+type Props = {
+  track: string;
+  raceNumber: number;
+};
+
+const useRunnerData = ({ track, raceNumber }: Props) => {
```

Comment:
would be good if we can add type for this hook.

Reply:
![image](https://user-images.githubusercontent.com/79682833/199492188-ca561b43-5c9b-49fb-9e87-1230165b2d2d.png)

due to `useSwr` accepting a generic Typescript already infers the return types for everything - so i don't think it's necessary - but happy to add if you think its critical